config/constants.ts

```diff
@@ -1,10 +1,33 @@
-let alphabet = '';
+const alphabet = [
+  'a',
+  'b',
+  'c',
+  'd',
+  'e',
+  'f',
+  'g',
+  'h',
+  'i',
+  'j',
+  'k',
+  'l',
+  'm',
+  'n',
+  'o',
+  'p',
+  'q',
+  'r',
+  's',
+  't',
+  'u',
+  'v',
+  'w',
+  'x',
+  'y',
+  'z'
+] as const;
 
-for (let i = 0; i < 26; i++) {
-  alphabet += String.fromCharCode(97 + i);
-}
```

Comment:
What's going to prevent someone from undoing this in the future - people try to act and code smart all the time, you know?

Reply:
Nothing at the moment. Do you want a test for the alphabet?

Reply:
It's not no obvious how to directly test this, but the last commit should make it clear that the blocklistedUsernames array is supposed to only contain literals.